api/turing/cluster/controller.go

```diff
@@ -680,7 +680,7 @@ func (c *controller) DeleteSparkApplication(ctx context.Context, namespace, appN
 
 func (c *controller) ApplyPodDisruptionBudget(
 	ctx context.Context,
-	namespace string,
+	_ string,
```

Comment:
is namespace still required even? can it be remove from the method arg

Reply:
Yeah that's true, it's not really needed anymore in this case. I'll remove it then; thanks for catching!